### PR TITLE
add social links

### PR DESCRIPTION
**Pull request by weichen6626**
#102

### CODE REVIEW

**Comment by pyxld-kris** [comment]
@weichen6626 This is great! Thank you. I made a small change to increase the size of the icons, what do you think?

File: src/components/common/Footer/Footer.js [hunk 1]
```diff
@@ -38,6 +38,28 @@ export default function Footer(props) {
...
         </Link>
       </div>
+      <div className={style.SocialMediaLink}>
+        <a
+          href="https://www.instagram.com/devlaunchers/"
+          class="fab fa-instagram"
+          target="_blank"
+        ></a>
+        <a
+          href=" https://www.linkedin.com/company/devlaunchers/"
```
Comment (chungthuang) [inline src/components/common/Footer/Footer.js:48]: Extra space here.

**Comment by chungthuang** [comment]
@weichen6626 great first PR!

**Pull request merged**